src/query/service/src/interpreters/access/management_mode_access.rs

```diff
@@ -62,6 +62,7 @@ impl ManagementModeAccess {
                             | RewriteKind::ShowFunctions
                             | RewriteKind::ShowUsers
                             | RewriteKind::ShowStages
+                            | RewriteKind::DescribeStage
```

Comment:
https://github.com/datafuselabs/databend/blob/59595a94453d8c7d31e52bc8c3fd84dce8e84a21/src/query/service/src/sql/planner/plans/mod.rs#L211-L227
Does this compile?

Reply:
It's outdated.